charts/kyverno/templates/crds.yaml

```diff
@@ -4,6 +4,7 @@ metadata:
   annotations:
     controller-gen.kubebuilder.io/version: v0.4.0
     config.kubernetes.io/index: '1'
+    internal.config.kubernetes.io/index: '1'
```

Comment:
What is this annotation? Do we need it?

Reply:
Shuting I dont think we need it, it was added when we run `make kustomize-crd`

Reply:
The weight field in preferredDuringSchedulingIgnoredDuringExecution is in the range 1-100. For each node that meets all of the scheduling requirements (resource request, RequiredDuringScheduling affinity expressions, etc.), the scheduler will compute a sum by iterating through the elements of this field and adding "weight" to the sum if the node matches the corresponding MatchExpressions. This score is then combined with the scores of other priority functions for the node. The node(s) with the highest total score are the most preferred.
https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#node-affinity

Reply:
I have removed all the related annotations in `crds.yaml`

Reply:
Hey @anushkamittal20 @realshuting Please ignore my comment. I misread it. This annotation `internal.config.kubernetes.io/index: '1'` has no relation with `weight`

Reply:
I have removed all these annotations in the file (there were 7) that came when I ran `make kustomize-crd`